### PR TITLE
Add storybook

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
Add storybook and stories for sandbox components

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 6.14.15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,9 @@
     "prepublishOnly": "yarn run lint && yarn run test && yarn run build",
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
-    "commit": "cz"
+    "commit": "cz",
+    "storybook:start": "start-storybook -p 6006",
+    "storybook:build": "build-storybook"
```
Comment (dangreen) [inline package.json:64]: We don't need to build this.

File: package.json [hunk 2]
```diff
@@ -59,7 +59,9 @@
     "prepublishOnly": "yarn run lint && yarn run test && yarn run build",
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
-    "commit": "cz"
+    "commit": "cz",
+    "storybook:start": "start-storybook -p 6006",
```
Comment (dangreen) [inline package.json:63]:
```suggestion
    "start:storybook": "start-storybook -p 6006",
```

File: .storybook/main.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const path = require('path')
+
+module.exports = {
+  stories: ['../stories/*.stories.(js|ts)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
```
Comment (dangreen) [inline .storybook/main.js:5]: > `'@storybook/addon-links', '@storybook/addon-essentials'`

Is it used in stories?

**Review by dangreen — CHANGES_REQUESTED** [review]
What about `.storybook/theme.js`?

File: .storybook/.babelrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./../.babelrc"
```
[the remark this answers is not in the document]
Comment (thabarbados) [reply .storybook/.babelrc:2]: removed

File: package.json (hunk 2 above)
Comment (thabarbados) [reply package.json]: fix

File: .storybook/main.js (hunk 3 above)
Comment (thabarbados) [reply .storybook/main.js]: `addon-links` not used. removed

**Review by dangreen — APPROVED** [review]

**Pull request merged**